src/gtest/test_httprpc.cpp

```diff
@@ -52,7 +60,7 @@ TEST(HTTPRPC, FailsWithBadAuth) {
     EXPECT_CALL(req, GetHeader("authorization"))
         .WillRepeatedly(Return(std::make_pair(true, "Basic spam:eggs")));
     EXPECT_CALL(req, GetPeer())
-        .WillRepeatedly(Return(CService("127.0.0.1:1337")));
+        .WillRepeatedly(Return(ResolveService("127.0.0.1:1337")));
```

Comment:
Interesting.. This helper is only used once, and it seems to do the exact same thing that the `CService` constructor used to do, before we started making the `CNetAddr` class family dumber. It's beyond the scope, but why is this helper test-only? It seems very useful.